src/include/uapi/ipc/header.h

```diff
@@ -103,6 +103,8 @@
 #define SOF_IPC_PM_CLK_GET			SOF_CMD_TYPE(0x005)
 #define SOF_IPC_PM_CLK_REQ			SOF_CMD_TYPE(0x006)
 #define SOF_IPC_PM_CORE_ENABLE			SOF_CMD_TYPE(0x007)
+#define SOF_IPC_PM_STATE_SET			SOF_CMD_TYPE(0x008)
```

Comment:
does this deprecate PM_CTX_SAVE/RESTORE then?

Reply:
for now PM_CTX_SAVE/RESTORE would be backward compatible to for D3 entry transition effectively, mid-term they would become deprecated
Could we switch to https://github.com/orgs/thesofproject/teams/sof-developers/discussions/ for a discussion on pm flows ?